src/Styles/BookRender.css

```diff
@@ -0,0 +1,117 @@
+.book-data {
+  display: flex;
+  flex-direction: row;
+  border: 1px solid rgb(197, 186, 186);
+  width: 90%;
+  margin-top: 30px;
+  padding: 1%;
+}
+
```

Comment:
- [ ] kindly use the zeplin blue color so they styling looks alike. `#0290ff`
- [ ] Great job. please use the zeplin color on the remove, edit and comment so they follow the original design
- [ ] kindly add a bottom margin so the add author and title can be clearly visible. also so it follows the original design
yours
![Screenshot (602)_LI](https://user-images.githubusercontent.com/65298048/202810580-4e881203-fe18-4ca7-96b4-36d55e101d8b.jpg)
zeplin
![Screenshot (601)_LI](https://user-images.githubusercontent.com/65298048/202810758-69f9e09d-47c0-4a9d-bf5c-388be033f0e3.jpg)